RCSwitch.cpp

```diff
@@ -310,9 +310,36 @@ char* RCSwitch::getCodeWordA(const char* sGroup, const char* sDevice, bool bStat
 }
 
 /**
+<<<<<<< HEAD
```

Comment:
This looks like you had a merge or rebase conflict and did not resolve it completely?